switchbot_ros/scripts/switchbot.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import requests, json
+from os import path
```

Comment:
```suggestion
from __future__ import print_function

import json
import os.path
import requests
```

keep alphabetical order and import one module in one line.